include/yaramod/types/regexp.h

```diff
@@ -505,24 +648,52 @@ class Regexp : public String
 	*                    ^
 	* @endcode
 	*/
-	const std::string& getSuffixModifiers() const
+	std::string getSuffixModifiers() const
 	{
-		return _suffixMods;
+		if (!_suffixMods)
+			return "";
+		else
+			return (*_suffixMods)->getPureText();
```

Comment:
Also why can't you do `return _suffixMods.value_or(std::string{});`

Reply:
Because the types differ: `_suffixMods`'s type is `std::optional<TokenIt>` and I need to access the `Token::getPureText()` method.